src/FSharp.Analyzers.SDK/TASTCollecting.fsi

```diff
@@ -11,12 +11,27 @@ module TASTCollecting =
         new: unit -> TypedTreeCollectorBase
 
         /// Overwriting this member hooks up a custom operation for a call of a member or function.
-        abstract WalkCall: range -> FSharpMemberOrFunctionOrValue -> FSharpExpr list -> unit
-        default WalkCall: range -> FSharpMemberOrFunctionOrValue -> FSharpExpr list -> unit
+        abstract WalkCall:
+            exprRange: range ->
```

Comment:
I can't help myself, but could we put the range as the last parameter?
That feels more consistent with the untyped syntax tree.
Totally nit, sorry again!